HotorNot/app/src/main/java/com/example/hotornot/MainActivity.kt

```diff
@@ -15,37 +15,38 @@ import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.view.menu.MenuBuilder
 import androidx.databinding.DataBindingUtil
 import androidx.navigation.NavController
+import androidx.navigation.Navigation
+import androidx.navigation.findNavController
 import androidx.navigation.fragment.NavHostFragment
 import androidx.navigation.fragment.findNavController
+import androidx.navigation.ui.AppBarConfiguration
 import androidx.navigation.ui.onNavDestinationSelected
 import androidx.navigation.ui.setupActionBarWithNavController
 import com.example.hotornot.databinding.ActivityMainBinding
+import com.example.hotornot.fragments.MotivationFragment
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMainBinding
     private lateinit var navController: NavController
     private lateinit var preferencesUtil: PreferencesUtil
+    private lateinit var appBarConfiguration: AppBarConfiguration
+    private var onPauseTime = 0
+    private var times = 600000
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         preferencesUtil = PreferencesUtil.getInstance(applicationContext)
-        binding = DataBindingUtil.setContentView(this,R.layout.activity_main)
-        loadingFragments()
-    }
-
-    private fun loadingFragments(){
```

Comment:
Change the name with better or directly writing in onCreate() because this method doing too many things